### PR TITLE
Filter vmem

### PR DESCRIPTION
**Pull request by stdname**
Added metric filters to filter out excessive metrics.
Configured such filter for vmem plugin to send only limited set of data

### CODE REVIEW

File: collectd.conf.tmpl [hunk 1]
```diff
@@ -87,6 +87,9 @@ LoadPlugin vmem
 
 LoadPlugin uptime
 
+Include "/etc/collectd.d/filtering_config/*.conf"
+Include "/etc/collectd.d/filtering_config/filtering.conf"
```
Comment (mdubbyap) [inline collectd.conf.tmpl]: these are not both required;  *.conf includes filtering.conf

**Comment by mdubbyap** [comment]
@ramjothikumar can we have this branch ran through your test framework?

File: collectd.conf.tmpl (hunk 1 above)
Comment (stdname) [inline collectd.conf.tmpl]: Nice catch. I'll remove it.

**Comment by stdname** [comment]
Request to run test suit:
https://signalfuse.atlassian.net/browse/FUSE-6311

**Comment by ramjothikumar** [comment]
Tested against ubuntu1204/1404/1504, centos6/7 and Amazon linux 2014.09/2015.03/2015.09

**Pull request merged**